src/script/interpreter.cpp

```diff
@@ -1070,6 +1071,8 @@ bool EvalScript(std::vector<std::vector<unsigned char> >& stack, const CScript&
                 break;
 
                 //////////////////////////////////////////////////////// qtum
+                case OP_SENDER:
```

Comment:
We should interpret OP_SENDER as OP_UNKNOWN and set the error to SCRIPT_ERR_BAD_OPCODE if height < QIP5Height, otherwise it is possible to cause a chain split between upgraded and non-upgraded nodes before the HF has taken place, by sending a specially crafted transaction containing OP_SENDER.

Reply:
The `op_sender` height is checked in `CheckOpSender`.
Added also new check in `EvalScript`.